.travis/linux_tengine_runner.sh

```diff
@@ -44,28 +44,29 @@ tengine_install() {
         return
     fi
 
-    wget https://openresty.org/download/openresty-1.15.8.2.tar.gz
-    tar zxf openresty-1.15.8.2.tar.gz
+    local OPENRESTY_VERSION="1.15.8.2"
+    wget https://openresty.org/download/openresty-${OPENRESTY_VERSION}.tar.gz
+    tar zxf openresty-${OPENRESTY_VERSION}.tar.gz
     wget https://codeload.github.com/alibaba/tengine/tar.gz/2.3.2
     tar zxf 2.3.2
     wget https://codeload.github.com/openresty/luajit2/tar.gz/v2.1-20190912
     tar zxf v2.1-20190912
     wget https://codeload.github.com/simplresty/ngx_devel_kit/tar.gz/v0.3.1
     tar zxf v0.3.1
 
-    rm -rf openresty-1.15.8.2/bundle/nginx-1.15.8
-    mv tengine-2.3.2 openresty-1.15.8.2/bundle/
+    rm -rf openresty-${OPENRESTY_VERSION}/bundle/nginx-1.15.8
```

Comment:
still has one: `/bundle/nginx-1.15.8`

Reply:
> still has one: `/bundle/nginx-1.15.8`

cause the openresty is bundled with that one..This is the least cost